### DISCUSSION TITLE
AdminLTE - Direct Chat Component

### DISCUSSION THREAD

**Discussion by GauravGoyal01** [ColorlibHQ/AdminLTE#4846] [General]
Hi,

It seems https://github.com/ColorlibHQ/AdminLTE is the bootstrap template which provide only UI components. 

Could someone please confirm "Direct Chat" component given in below AdminLTE only an UI component or its a fully functional chat/Messaging system?

https://adminlte.io/docs/3.0/components/direct-chat.html

Please confirm.....

---
Thanks

**Comment by lenamtl** [comment]
Yes this is only a UI, you will need to code it.
